server/chat-commands/admin.ts

```diff
@@ -104,6 +104,25 @@ async function rebuild(context: CommandContext) {
 
 
 export const commands: ChatCommands = {
+	potd(target, room, user) {
+		this.canUseConsole();
+		target = toID(target);
```

Comment:
What's this for?

Reply:
`getSpecies` already has an ID conversion, and the only other place `target` is used is an error message that doesn't need to be in ID form.

```suggestion
```